test/bootstrap.php

```diff
@@ -15,134 +16,24 @@
 
 // Attempt to start our own redis instance for tesitng.
 exec('which redis-server', $output, $returnVar);
-if($returnVar != 0) {
-	echo "Cannot find redis-server in path. Please make sure redis is installed.\n";
-	exit(1);
+if ($returnVar != 0) {
+    echo "Cannot find redis-server in path. Please make sure redis is installed.\n";
+    exit(1);
 }
 
 exec('cd ' . TEST_MISC . '; redis-server ' . REDIS_CONF, $output, $returnVar);
 usleep(500000);
-if($returnVar != 0) {
-	echo "Cannot start redis-server.\n";
-	exit(1);
+if ($returnVar != 0) {
+    echo "Cannot start redis-server.\n";
+    exit(1);
 
 }
 
 // Get redis port from conf
 $config = file_get_contents(REDIS_CONF);
-if(!preg_match('#^\s*port\s+([0-9]+)#m', $config, $matches)) {
-	echo "Could not determine redis port from redis.conf";
-	exit(1);
+if (!preg_match('#^\s*port\s+([0-9]+)#m', $config, $matches)) {
```

Comment:
I've seen this same thing done 3 different times now--maybe it'd be good to make this a function?